dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/utils/ReferenceConfigCache.java

```diff
@@ -149,5 +202,6 @@ public String toString() {
 
     public static interface KeyGenerator {
         String generateKey(ReferenceConfig<?> referenceConfig);
+        String generateKey(String version,String url,String name);
```

Comment:
Why the parameters are version, url and name? not interfaceName, group and version?